heron/cli/src/python/execute.py

```diff
@@ -106,6 +111,10 @@ def heron_pex(topology_pex, topology_class_name, tmp_dir):
   try:
     pex_loader.load_pex(topology_pex)
     topology_class = pex_loader.import_and_get_class(topology_pex, topology_class_name)
+    if opts.get_config("cmdline.topology.initial.state") == "PAUSED":
```

Comment:
use Log.debug("Topology deployment status %s" % (opt.get_config(...))

so that it is displayed even in the case of active vs paused.